modules/client/package.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
Could we add some test to check `parseToken`?

Reply:
done ✨